pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -205,7 +205,14 @@ func activeQComp(podInfo1, podInfo2 interface{}) bool {
 	pInfo2 := podInfo2.(*framework.PodInfo)
 	prio1 := pod.GetPodPriority(pInfo1.Pod)
 	prio2 := pod.GetPodPriority(pInfo2.Pod)
-	return (prio1 > prio2) || (prio1 == prio2 && pInfo1.Timestamp.Before(pInfo2.Timestamp))
+
+	// To make PriorityQueue near FIFO for same priority Pods, i.e. <5% Pods cut
+	// in line, we should use Pod creation time instead of the last queued time by
+	// scheduler to sort Pod for the same priority.
+	ctime1 := pInfo1.Pod.CreationTimestamp.Time
```

Comment:
`ctime` can be renamed as `creationTime` to look more meaningful ?

Reply:
Thanks! Updated